src/languages/en.js

```diff
@@ -123,7 +123,7 @@ export default {
         attachmentTooLarge: 'Attachment too large',
         sizeExceeded: 'Attachment size is larger than 24 MB limit.',
         attachmentTooSmall: 'Attachment too small',
```

Comment:
I figure we should do the same for the rest as well?

Reply:
@youssef-lr I checked those as well but they are used in a title, not in a modal/error body https://github.com/Expensify/App/blob/aff83ef1895d48603f9610bf56bd06722b8e10c2/src/components/AttachmentModal.js#L169

https://github.com/Expensify/App/blob/aff83ef1895d48603f9610bf56bd06722b8e10c2/src/components/AttachmentModal.js#L178

We dont want dot there.

Reply:
ok nice